### PR TITLE
Added content in files

### PR DESCRIPTION
**Pull request by nitibilkhu**
Add Header and footer in all files

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -5,6 +5,8 @@ body {
 h1 {
   color: white;
   text-align: center;
+  color: yellow;
```
Comment (nitibilkhu) [inline main.css:8]: Change color white to red

File: main.css [hunk 2]
```diff
@@ -5,6 +5,8 @@ body {
...
+  text-align: left;
 }
 
 p {
```
Comment (nitibilkhu) [inline main.css:12]: Add more styling to css file

File: index.js [hunk 3]
```diff
@@ -1,2 +1,3 @@
 document.get("Hello World");
 document.get("Hello World to js Programming");
```
Comment (nitibilkhu) [inline index.js:2]: Add 2 more document.get("");

File: index.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <body>
-
+Hello world to HTML
```
Comment (nitika807) [inline index.html:4]: Can you add some description of the provided headings.

File: main.css [hunk 5]
```diff
@@ -5,6 +5,8 @@ body {
 h1 {
```
Comment (nitika807) [inline main.css:5]: Can you define some style for other headings ?

**Review by nitika807 — CHANGES_REQUESTED** [review]
I have added some requests in the code. Please look into this if those can be catered to ?

**Pull request merged**